services/auth/token/lambdas/generateToken.js

```diff
@@ -0,0 +1,13 @@
+import { signToken } from '../helpers/token';
+
+export const main = async event => {
```

Comment:
main = async event => {...
main = async (event) => {...
according to my linting rules